src/main/resources/db/changelog/changes/v1.1.0/sql/entity-type-definitions/insert-organization-definition.sql

```diff
@@ -0,0 +1,284 @@
+INSERT INTO entity_type_definition (id, derived_table_name, definition)
+    VALUES ('837f262e-2073-4a00-8bcc-4e4ce6e669b3','drv_organization_details', '{
+             "id": "837f262e-2073-4a00-8bcc-4e4ce6e669b3",
+             "name":"drv_organization_details",
+             "private" : false,
+             "fromClause" : "src_organizations AS org LEFT JOIN LATERAL jsonb_array_elements(org.jsonb->''accounts'') AS accounts(account_data) ON true",
```

Comment:
By chance, are there any indexes in the organizations table that we can use in filterValueGetters for these columns?

Reply:
@mweaver-ebsco good point totally forgot about those. will check and add